specs/powerfulfeatures/index.src.html

```diff
@@ -313,16 +313,16 @@ <h4 id="legacy-example">Example: Geolocation</h4>
   <ol>
     <li>
       <a lt="modify a specification">Modify</a> the specification to include
-      checks against [[#settings-privileged] before executing the algorithms for
+      checks against [[#settings-secure] before executing the algorithms for
       {{getCurrentPosition()}} and {{watchPosition()}}.
 
-      If [[#settings-privileged]] returns <code>Not Privileged</code>, then
+      If [[#settings-secure]] returns <code>Not Secure</code>, then
       the algorithms should be aborted, and the <var>errorCallback</var> invoked
       with a <code>code</code> of <code>PERMISSION_DENIED</code>.
     </li>
     <li>
       User agents should announce clear intentions to disable the API for
-      unprivileged contexts on a specific date, and warn developers accordingly
+      unsecure contexts on a specific date, and warn developers accordingly
```

Comment:
Or non-secure, if there's a good reason for that term? Basically, we should be consistent.